packages/editor-ui/src/components/RunData.vue

```diff
@@ -1360,6 +1360,9 @@ defineExpose({ enterEditMode });
 			<N8nText v-n8n-html="hint.message" size="small"></N8nText>
 		</N8nCallout>
 
+		<div v-if="hasNodeRun">
+			<slot name="panel-callout-info"></slot>
```

Comment:
The slot here allows us to inject components for different panels, but the callout here could be reused for different features and panels. Because we might want to reuse this in the future, it would be best to add the Callout component here and the message as a prop. 
Also this would get rid of the unnecessary empty div in the test/render. 

```
<N8nCallout v-if="props.calloutMessageKey" "props.:key="calloutMessageKey" />
```